lua/apg/cl_menu.lua

```diff
@@ -204,7 +204,7 @@ local function openMenu( len )
     saveButton:SetSize(72,16)
     saveButton:SetText('')
     saveButton.DoClick = function()
-        local settings = APG
+        settings = APG

```

Comment:
Why isn't this meant to be local? Seems like it could be a local var with no issues. This is also conflicting with a variable in the outer namespace.

Reply:
Reverted, don't remember the reason why I did it,  did this commit at like 4 in the morning :P